dashboard/app/models/pd/application/principal_approval_application.rb

```diff
@@ -160,8 +140,6 @@ def self.filtered_labels(course, status = 'unreviewed')
 
     def additional_text_fields
       [
-        [:committed_to_master_schedule],
-        [:replace_course, TEXT_FIELDS[:dont_know_explain], :replace_course_other],
         [:do_you_approve],
         [:contact_invoicing],
         [:contact_invoicing_detail]
```

Comment:
Same question here for these two fields as above?

Reply:
Removed! And found another place I forgot to remove these fields in this file!